lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1156,8 +1282,14 @@ class TextEditingChannel {
         break;
 
       case 'TextInput.finishAutofillContext':
-        // TODO(nurhan): Handle saving autofill information on web.
-        // https://github.com/flutter/flutter/issues/59378
+        final bool saveForm = call.arguments as bool;
+        // Close the text editing connection. Form is finalizing.
+        implementation.sendTextConnectionClosedToFrameworkIfAny();
```

Comment:
Why close the connection here?

The framework `AutofillGroup` disposal will send 'TextInput.finishAutofillContext' message, But dispose is an asynchronous process, and the wrong client ID may be closed here.

Reply:
Issue reported at https://github.com/flutter/flutter/issues/71476